packages/lib/src/file-input/FileItem.tsx

```diff
@@ -44,17 +45,18 @@ const FileItem = ({
                 <DxcIcon icon="filled_error" />
               </ErrorIcon>
             )}
-            <DeleteFileAction
-              onClick={() => {
-                onDelete(fileName);
-              }}
-              type="button"
-              title={translatedLabels.fileInput.deleteFileActionTitle}
-              aria-label={translatedLabels.fileInput.deleteFileActionTitle}
-              tabIndex={tabIndex}
-            >
-              <DxcIcon icon="close" />
-            </DeleteFileAction>
+            <DxcTooltip label={translatedLabels.fileInput.deleteFileActionTitle}>
+              <DeleteFileAction
```

Comment:
Why this isn't a `DxcActionIcon`? Maybe we can use it here!

Reply:
Good point.